.github/workflows/spectesting.yml

```diff
@@ -0,0 +1,58 @@
+on:
+  push:
+    branches:
+      - 'spec-test-*'
+    paths-ignore:
+      - 'CHANGELOG.md'
+      - 'website/**'
+
+jobs:
+  setup-packet:
+    if: github.repository == 'hashicorp/vagrant-blackbox'
+    runs-on: self-hosted
+    name: Build Packet Instance
+    steps:
+      - name: Create packet instance
+        run: .ci/spec/create-packet.sh
+        working-directory: ${{github.workspace}}
+
+  setup-hosts:
+    if: github.repository == 'hashicorp/vagrant-blackbox'
+    runs-on: self-hosted
+    name: Vagrant-Spec Start Hosts
+    needs: setup-packet
+    steps:
+      - name: Code Checkout
+        uses: actions/checkout@v1
+      - name: Run Tests with host ${{ matrix.host_os }} using guest ${{ matrix.guest_os }}
```

Comment:
Needs matrix strategy defined, but if testing initially without, will need to remove the matrix usage within the name content.

Reply:
Oops, I had it originally, but removed it when I started breaking out the workflow into multiple jobs. I'll put it back. :+1: